frontend/src/components/StatusTag.vue

```diff
@@ -200,7 +200,7 @@ export default {
         return 'error'
       }
       if (this.isUnknown) {
-        return 'grey lighten-1'
+        return 'grey lighten'
```

Comment:
There is no class `lighten`

Reply:
Sorry, wrong class name, should be `grey` only. We don't want to lighten the grey (as your last feedback was to make the grey darker). I fixed this.